UI/css/addentry.css

```diff
@@ -0,0 +1,2300 @@
+body {

```

Comment:
pls avoid using camel casing for file names

Reply:
how is it that you have 2207 lines of css written within a day?

Reply:
I have so many media queries for the different screen sizes. What I did was write the styling for one screen size, copy the styling and add it to another screen size, and if adjustments have to be made, I effect same.